.konchrc

```diff
@@ -27,3 +24,11 @@ konch.config(
 
 if Path(".konchrc.local").exists():
     konch.use_file(".konchrc.local", trust=True)
+
+
+def setup():
```

Comment:
Remove these functions, as they're unrelated to the fix.

Reply:
These functions were being used for troubleshooting, they were left thinking they were helpful, but definitely not needed for the fix, I will not include in new PRs.